core/shared/src/test/scala/zio/prelude/DebugSpec.scala

```diff
@@ -192,6 +192,16 @@ object DebugSpec extends DefaultRunnableSpec {
         testM("testTrait")(check(genTestTrait) { c =>
           assert(c.debug.render(Renderer.Full))(equalTo(s"DebugSpec.${c.getClass.getSimpleName.init}"))
         })
+      ),
+      suite("DebugInterpolator")(
+        test("String") {
+          val name = "World"
+          assert(dbg"Hello, my name is $name.")(equalTo("Hello, my name is \"World\"."))
```

Comment:
:point_up: this is the test :smile_cat: `s"..."` doesn't add `"`s around strings, for example.

Reply:
Ah, too subtle for me! 😃